framework/console/controllers/QueueController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\console\controllers;
+
+use Yii;
+use yii\console\Controller;
+
+/**
+ * Runs the queue listener.
+ *
+ * @author Alexander Kochetov <creocoder@gmail.com>
+ * @since 2.0.5
+ */
+class QueueController extends Controller
```

Comment:
> you think daemon on php is impossible

Just where I have told you daemon on PHP is impossible?
All I say: there is no need for predefined controller for the queue processing. Developer should decide, if he want it inside console command or inside daemon or anywhere else.
That is why I said at the begining:

> Such class seems to be redundant.